### ISSUE TITLE
Freecad rule removed

### ISSUE THREAD

**Issue by guiweber** [just-install/just-install-updater#36]
The freecad rule has been removed since their x32 and x64 versions are not kept at the same version number, which makes the update fail. Lets see if they fix this later or find a way to allow it.

```
"freecad": {
    "url": "https://www.freecadweb.org/wiki/Download",
    "updater": {
      "x86": {
        "rule_type": "css-link",
        "selector": "a.external",
        "filter": "WIN-x86"
      },
      "x86_64": {
        "rule_type": "css-link",
        "selector": "a.external",
        "filter": "WIN-x64"
      }
    },
    "version_extractor": "FreeCAD.([0-9]+(?:\\.[0-9]+)*[a-z]?).*\\.exe$"
  },
```

**Comment by cyagon** [comment]
freecad is using github releases to publish their software: https://github.com/FreeCAD/FreeCAD/releases
there are also posting their latest releases in their wiki: https://www.freecadweb.org/wiki/Download#Stable_FreeCAD_installers

maybe this can solve the problem. i will have a look at it when i have time.

![unbenannt](https://user-images.githubusercontent.com/32183089/35829438-2627c422-0ac3-11e8-9613-8ca8720f5331.PNG)

**Comment by guiweber** [comment]
Yes, I am aware of those. But if you take a look you'll notice that the latest x86 and x64 versions numbers are not the same, which is something that Just-Install doesn't support... On Github the latest release doesn't even have x86 binaries.
